src/components/MainNavigation.tsx

```diff
@@ -0,0 +1,48 @@
+import * as React from 'react'
+
+import Navigation from './Navigation'
+
+
+interface MainNavigationPro {
```

Comment:
You don't necessarily have to define empty prop interfaces. Just pass {} as props